README.md

```diff
@@ -1 +1,150 @@
-# hospital_system
\ No newline at end of file
+<center>
+<h1>Clinic database: Create a database based on a diagram</h1>
+</center>
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Clinic database] <a name="about-project"></a>
+This repository includes files with plain SQL that can be used to create a database respecting the provided clinic database_diagram:
+
+>- Use [schema_based_on_diagram.sql](./schema_based_on_diagram.sql) to create all tables.
+
+<a name="readme-top"></a>
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **[Create database]**
+- **[Create table]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To get a local copy up and running, follow these steps.
+
+
+### Prerequisites
+
+In order to run this project you need:
+<center>
+<div>
+<a href="https://www.postgresql.org/download/">
+<img src="image.png" width="20%"/>
+</a>
+</div>
+</center>
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/Graciano1997/hospital_system.git
+```
+-
+
+### Install
+
+Install this project with:
+
+Example command:
+
+```sh
+  copy the schema_based_on_diagram.sql scripts and run in Query tool 
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author**
+
+- GitHub: [@Graciano1997](https://github.com/Graciano1997/)
+👤 **Author**
+
+- GitHub: [@Maurice](https://github.com/coker2019/)
+
```

Comment:
- [ ] Since it is a group project, kindly add information for all contributors :+1: